testsuite/integration-arquillian/servers/auth-server/jboss/common/crossdc/cross-dc-setup.cli

```diff
@@ -0,0 +1,133 @@
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=offlineSessions/store=custom:add( \
+    class=org.keycloak.models.sessions.infinispan.remotestore.KeycloakRemoteStoreConfigurationBuilder, \
+    passivation=false, \
+    fetch-state=false, \
+    purge=false, \
+    preload=false, \
+    shared=true \
+)
+
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=offlineSessions/store=custom:write-attribute( \
+    name=properties, value={ \
+        remoteCacheName=offlineSessions, \
+        useConfigTemplateFromCache=work, \
+        sessionCache=true \
+    } \
+)
+
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=offlineSessions:write-attribute(name=statistics-enabled,value=true)
+
+echo ** Update distributed-cache loginFailures element **
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=loginFailures/store=custom:add( \
+    class=org.keycloak.models.sessions.infinispan.remotestore.KeycloakRemoteStoreConfigurationBuilder, \
+    passivation=false, \
+    fetch-state=false, \
+    purge=false, \
+    preload=false, \
+    shared=true \
+)
+
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=loginFailures/store=custom:write-attribute( \
+    name=properties, value={ \
+        remoteCacheName=loginFailures, \
+        useConfigTemplateFromCache=work, \
+        sessionCache=true \
+    } \
+)
+
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=loginFailures:write-attribute(name=statistics-enabled,value=true)
+
+echo ** Update distributed-cache actionTokens element **
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=actionTokens/store=custom:add( \
+    class=org.keycloak.models.sessions.infinispan.remotestore.KeycloakRemoteStoreConfigurationBuilder, \
+    passivation=false, \
+    fetch-state=false, \
+    purge=false, \
+    preload=true, \
+    shared=true \
+)
+
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=actionTokens/store=custom:write-attribute( \
+    name=properties, value={ \
+        remoteCacheName=actionTokens, \
+        useConfigTemplateFromCache=work, \
+        sessionCache=false \
+    } \
+)
+
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=actionTokens:write-attribute(name=statistics-enabled,value=true)
+
+echo ** Update distributed-cache authenticationSessions element **
+/subsystem=infinispan/cache-container=keycloak/distributed-cache=authenticationSessions:write-attribute(name=statistics-enabled,value=true)
+
+echo *** Enable debug logging ***
+/subsystem=logging/logger=org.keycloak.cluster.infinispan:add
+/subsystem=logging/logger=org.keycloak.cluster.infinispan:write-attribute(name="level", value="DEBUG")
+
+/subsystem=logging/logger=org.keycloak.connections.infinispan:add
+/subsystem=logging/logger=org.keycloak.connections.infinispan:write-attribute(name="level", value="DEBUG")
+
+/subsystem=logging/logger=org.keycloak.models.cache.infinispan:add
```

Comment:
This can be simplified to `/subsystem=logging/logger=org.keycloak.models.cache.infinispan:add(level=DEBUG)`